recipes-bsp/imx-mkimage/imx-boot_0.2.bb

```diff
@@ -121,7 +125,9 @@ compile_mx8x() {
 }
 do_compile() {
     compile_${SOC_FAMILY}
-    # mkimage for i.MX8
+    if ${DEPLOY_OPTEE}; then
+        cp ${DEPLOY_DIR_IMAGE}/tee.bin ${BOOT_STAGING}
+    fi
```

Comment:
This should be done on do_install.

Reply:
I guess this includes all of the compile_mx* functions as well?

Reply:
compile is fine to be on compile, but installation should be on the other task.

Reply:
The copying needs to be done here since the make is done with BOOT_STAGING as input. All the copying is part of setting up the compilation.